magstock/__init__.py

```diff
@@ -45,6 +52,10 @@ class Attendee:
     def food_cost(self):
         return c.FOOD_PRICE if self.purchased_food else 0
 
+    @property
+    def auto_food(self):
+        return self.badge_type in [c.STAFF_BADGE, c.GUEST_BADGE]
```

Comment:
You're right, this is definitely a situation where the same logic is implemented in two places.

The reason for this is efficiency.  The only way to use this property in the ``SessionMixin`` class would be to actually load all of the results into memory so the property could be checked.  We want to be able to do a ``.count()`` on the query, which means we end up being forced into defining the same thing in two places for efficiency.

Hope that makes sense.